paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -380,14 +391,23 @@ class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
                           "Operator DNNL Conv must use CPUPlace"));
     bool is_INT8 =
         std::is_same<T, int8_t>::value || std::is_same<T, uint8_t>::value;
+    bool is_BFLOAT16 = ctx.Attr<std::string>("mkldnn_data_type") == "bfloat16";
+    auto residual_param = ctx.Input<Tensor>("ResidualData");
+    bool fuse_residual_conn = ctx.Attr<bool>("fuse_residual_connection");
+    std::string fuse_activation = ctx.Attr<std::string>("fuse_activation");
+    bool force_fp32_output = ctx.Attr<bool>("force_fp32_output");
+
     if (!is_INT8) {
-      ComputeFP32<float>(ctx);
+      auto dst_dt =
```

Comment:
This could be simplified:
```
auto dst_dt = GetDstType(is_INT8, is_BFLOAT16, force_fp32_output, fuse_activation,
                     fuse_residual_conn, residual_param);
if (!is_INT8) {
   if (dst_dt = ...) ...
} else {
   if (dst_dt = ...) ...
}
```

Reply:
Done